src/fontra/views/editor/panel-reference-font.js

```diff
@@ -378,8 +442,13 @@ export default class ReferenceFontPanel extends Panel {
     this.controller.addKeyListener("fontList", (event) =>
       this._fontListChangedHandler(event)
     );
+    this.controller.addKeyListener("supportedLanguages", () => {
```

Comment:
I don't think we should put supportedLanguages on the model, as this will also make it save to localStorage.

What works equally well is to delete the key listener, and call setSupportedLanguages instead of this line:
```js
 this.model.supportedLanguages = fontItem.supportedLanguages;
```

Reply:
This is strange. The reference font list wasn't loading with the previously selected font. Now it does. This is why I cached it in localstorage.

I will apply your suggestion.

Reply:
It happens when multiple tabs of fontra is open. I applied your suggestion.

Reply:
What is the status of this? I'm not sure I understand the issue, if there still is one. Can you give concrete steps about what to do and what to observe?